src/vs/base/common/uri.ts

```diff
@@ -48,10 +48,8 @@ function _validateUri(ret: URI, _strict?: boolean): void {
 			if (!_singleSlashStart.test(ret.path)) {
 				throw new Error('[UriError]: If a URI contains an authority component, then the path component must either be empty or begin with a slash ("/") character');
 			}
-		} else {
```

Comment:
Can you please revert the changes in this file too. I don't think they are part of this PR